lib/src/main/java/com/kirkbushman/araw/RedditApi.kt

```diff
@@ -240,6 +240,7 @@ interface RedditApi {
     fun submission(
         @Query("id") submissionId: String,
         @Query("raw_json") rawJson: Int? = null,
+        @Query("sr_detail") srDetail: Int = 1,
```

Comment:
It's better if we make this field nullable, and the default parameter null. In this case Retrofit will not set this by default. And the user will opt-in if necessary.